sslSniffer/sslSnifferTest/snifftest.c

```diff
@@ -469,17 +478,21 @@ static void show_usage(void)
     printf("\t./snifftest dump pemKey [server] [port] [password]\n");
 }
 
-
-#ifdef WOLFSSL_ASYNC_CRYPT
-
 typedef struct SnifferPacket {
     byte* packet;
     int   length;
     int   lastRet;
     int   packetNumber;
+#ifdef THREADED_SNIFFTEST
+    struct SnifferPacket* next;
+    struct SnifferPacket* prev;
+    int    valid;
```

Comment:
What is `valid` used for? Seems like a workaround for something. Can you just check if `packet == NULL`? Consider better name if really required or making volatile.

Reply:
Yeah it's a workaround for the threads that might not receive packets. Renamed to `placeholder`.